<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_diagnosisRepairSystemRetrieveData</class>
 <widget class="QDialog" name="menu_diagnosisRepairSystemRetrieveData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>182</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>332</width>
    <height>182</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>332</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QProgressBar" name="currentProgress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="statusDirLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="detailsBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="overallProgress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="currentLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Progress :</string>
   </property>
  </widget>
  <widget class="QLabel" name="overallLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Overall Progress : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="detailsButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
